tdrs-backend/tdpservice/search_indexes/documents/tribal.py

```diff
@@ -1 +1,186 @@
 """Elasticsearch document mappings for TRIBAL submission models."""
+
+from django_elasticsearch_dsl import Document
+from django_elasticsearch_dsl.registries import registry
+from ..models.tribal import Tribal_TANF_T1, Tribal_TANF_T2, Tribal_TANF_T3
+from .document_base import DocumentBase
+
+@registry.register_document
+class Tribal_TANF_T1DataSubmissionDocument(DocumentBase, Document):
+    """Elastic search model mapping for a parsed TANF T1 data file."""
+
+    class Index:
+        """ElasticSearch index generation settings."""
+
+        name = 'tribal_tanf_t1_submissions'
+        settings = {
+            'number_of_shards': 1,
+            'number_of_replicas': 0,
+        }
+
+    class Django:
+        """Django model reference and field mapping."""
+
+        model = Tribal_TANF_T1
+        fields = [
+            'RecordType',
+            'RPT_MONTH_YEAR',
+            'CASE_NUMBER',
+            'COUNTY_FIPS_CODE',
+            'STRATUM',
+            'ZIP_CODE',
+            'FUNDING_STREAM',
+            'DISPOSITION',
+            'NEW_APPLICANT',
+            'NBR_FAMILY_MEMBERS',
+            'FAMILY_TYPE',
+            'RECEIVES_SUB_HOUSING',
+            'RECEIVES_MED_ASSISTANCE',
+            'RECEIVES_FOOD_STAMPS',
+            'AMT_FOOD_STAMP_ASSISTANCE',
+            'RECEIVES_SUB_CC',
+            'AMT_SUB_CC',
+            'CHILD_SUPPORT_AMT',
+            'FAMILY_CASH_RESOURCES',
+            'CASH_AMOUNT',
+            'NBR_MONTHS',
+            'CC_AMOUNT',
+            'CHILDREN_COVERED',
+            'CC_NBR_MONTHS',
+            'TRANSP_AMOUNT',
+            'TRANSP_NBR_MONTHS',
+            'TRANSITION_SERVICES_AMOUNT',
+            'TRANSITION_NBR_MONTHS',
+            'OTHER_AMOUNT',
+            'OTHER_NBR_MONTHS',
+            'SANC_REDUCTION_AMT',
+            'WORK_REQ_SANCTION',
+            'FAMILY_SANC_ADULT',
+            'SANC_TEEN_PARENT',
+            'NON_COOPERATION_CSE',
+            'FAILURE_TO_COMPLY',
+            'OTHER_SANCTION',
+            'RECOUPMENT_PRIOR_OVRPMT',
+            'OTHER_TOTAL_REDUCTIONS',
+            'FAMILY_CAP',
+            'REDUCTIONS_ON_RECEIPTS',
+            'OTHER_NON_SANCTION',
+            'WAIVER_EVAL_CONTROL_GRPS',
+            'FAMILY_EXEMPT_TIME_LIMITS',
+            'FAMILY_NEW_CHILD'
```

Comment:
I don't know our standards on this, but here we don't have a trailing comma and on line 143, we do.